app/views/offers/edit.html.erb

```diff
@@ -13,8 +13,7 @@
     <%= f.input :end_date %>
     <%= f.input :photos, as: :file, input_html: { multiple: true } %>
     <%= f.button :submit, class:"btn-smpl-form" %>
+    <%= link_to "cancel", overview_path, :class=>"btn-ghost" %>
```

Comment:
:class => ... is old syntax; best to write in the new way / class: '...'

Reply:
done